libexec/integration-test.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @file This script starts dev server and runs cypress tests.
+ */
+
+const { spawn } = require('child_process')
+spawn('npx', ['http-server', 'dist'],{shell:true})
```

Comment:
Minor style suggestion — my mind is so used to separating imports from code that uses imports, that adding a single space makes this much easier for me to scan! 😅 

```suggestion
const { spawn } = require('child_process')

spawn('npx', ['http-server', 'dist'], {shell: true})
```